modules/medianetBidAdapter.js

```diff
@@ -1,9 +1,12 @@
 import { registerBidder } from 'src/adapters/bidderFactory';
 import * as utils from 'src/utils';
+import { config } from 'src/config';
 
 const BIDDER_CODE = 'medianet';
 const BID_URL = 'https://prebid.media.net/rtb/prebid';
 
+$$PREBID_GLOBAL$$.medianetGlobals = {};
```

Comment:
This is ok since no public api functions are being invoked but we encourage adapters to use `$$PREBID_GLOBAL$$` as little as possible. `localStorage` might suit this use case without needing to use the pbjs global